src/styles/quarks/_quarks.css-properties.scss

```diff
@@ -15,6 +15,14 @@
   --theme1-300:              #{$endeavour-300};
   --theme1-200:              #{$endeavour-200};
   --theme1-100:              #{$endeavour-100};
+  --theme1-050:              #{$endeavour-050};
+
+  --theme2-600:              #{$monza-600};
+  --theme2-500:              #{$monza-500};
+  --theme2-400:              #{$monza-400};
+  --theme2-300:              #{$monza-300};
+  --theme2-200:              #{$monza-200};
+  --theme2-100:              #{$monza-100};
```

Comment:
Nog geen `monza-50`?

Reply:
Ik heb dezelfde vraag ook gesteld aan Jan Nikolaas. `monza-100` is reeds zeer licht, dus daarom is er nog geen `monza-050` gekozen. 🙂